server/models/searches.js

```diff
@@ -2,86 +2,36 @@ var mongoose = require('mongoose');
 	
 	
 var searchSchema = new mongoose.Schema({	
+    ids: {
+		type: [String]
+	},
     names: {
-		type: [[name]]
+		type: {
+			value: String,
+    		label: String,
+    		First: String,
+    		Last: String,
+		},
 	},
 	services: { 
-		type: [[service]]
+		type: [String]
 	},
 	location: {
-		type: {locationSchema}
-	},
-	time: {
-		type: {time}
-	},
-	timeOfDay: {
-		type: {timeOfDay}
-	},
-	numSessions: {
-		type: {numSession}
-	}
-}, { collection: 'searches' });
-
-var name = new mongoose.Schema({
-	first: {
-		type: String
-	},
-	last: {
 		type: String
 	},
-	label: {
-		type: String
-	},
-	value: {
-		type: String
-	}
-});
-
-var service = new mongoose.Schema({
-	label: {
-		type: String
-	},
-	value: {
-		type: String
-	}
-});
-
-var locationSchema = new mongoose.Schema({
-	label: {
+	time: {
 		type: String
 	},
-	value: {
+	timeOfDay: {
 		type: String
-	}
-});
-
-var numSession = new mongoose.Schema({
-	label: {
-		type: Number
 	},
-	value: {
+	numSessions: {
```

Comment:
Would [number](https://mongoosejs.com/docs/schematypes.html#numbers) work better for this?

Reply:
Yes, it would.